src/Sulu/Bundle/LocationBundle/Resources/js/package.json

```diff
@@ -5,8 +5,8 @@
     "repository": "https://github.com/sulu/sulu.git",
     "dependencies": {
         "classnames": "^2.2.5",
-        "leaflet": "^1.5.1",
-        "react-leaflet": "^3.2.0"
+        "leaflet": "^1.8",
+        "react-leaflet": "^4.0.1"
```

Comment:
the `react-leaflet` changelog sounds like we cannot use `whenCreated` here anymore: https://github.com/sulu/sulu/blob/03342c589016815f3913ae1aff1c711788681439/src/Sulu/Bundle/LocationBundle/Resources/js/containers/Location/Location.js#L112

Reply:
https://github.com/PaulLeCam/react-leaflet/releases/tag/v4.0.0